components/automate-ui/src/app/pages/top-nav-landing/top-nav-landing.component.html

```diff
@@ -0,0 +1,14 @@
+<app-landing [routePerms]="routeList" [onNotFound]="SetPageCoverage"></app-landing>
```

Comment:
"Call" the super-class to do permission-based routing.
The rest of the content of the page is what displays when there are no permissions--because we have not routed anywhere else.